packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -371,7 +388,11 @@ async function launchDesktopApp(app: OfficeApp, manifest: ManifestInfo, document
       );
     }
     await launchApp(app, await getOutlookExePath());
-  } else {
+  } else if (manifestPath.endsWith(".json"))
```

Comment:
There a couple of things to consider here
- Combine this with the outlook case
- Give that outlook xml manifests are also getting sideloaded via the service now that check is probably not needed anymore
- What is the xml sideloading story for WXP going forward (Outlook had to move to deploying xml via the service)?

Reply:
Hi Darren, this draft changes I made here is for the purpose of making minimum changes to existing pipeline first. According to your suggestions, we will refactor and combine Outlook cases. For the sideload of XML, I saw a TAOS which can sideload xml (https://titles.msit.mos.microsoft.com/dev/v1/users/packages/addins), you mean we can change the sideload of xml from embedded "web-extension" to TAOS now?

Reply:
I'm saying that was done because Outlook needed it . . . but the Outlook case for xml was already different for xml.  Outlook doesn't have a file to open . . . so the add-ins were actually referenced in exchange and Outlook got them from there . . . so sideloading was to exchange.  Syncing exchange with the new service (which supposed to be the new source of truth) was delayed which didn't work for the developer experience.  So they added a sideload xml manifest directly to the new service instead of indirectly through exchange (and Outlook was going to remove the call to get add-ins from exchange anyway).

I'm not sure what the plan is for WXP.  They have add-in references in files (by id, not be manifest itself).  The hosts are able to load xml manifests locally via the registry.  Json manifests cannot be loaded locally because the hosts need the app definition instead of the direct manifest so they will talk to the new service for that.  

Will WXP support file references for json manifest add-ins?  I don't know why they would remove that support.  If it's still there is this change even needed (loading can still be triggered via file)?

Will WXP remove the xml manifest loading from the registry when they switch to loading from the new service?  Do so would require sideloading xml to the new service, but that is not what this change is targeting.